inc/class-upgrade.php

```diff
@@ -1161,4 +1164,77 @@ public function migrate_og_settings_from_social_to_titles() {
 		\update_option( 'wpseo_social', $wpseo_social );
 		\update_option( 'wpseo_titles', $wpseo_titles );
 	}
+
+	/**
+	 * Migrates the frontpage social settings to the titles options.
```

Comment:
This summary appears to be a copy-paste from the existing method `migrate_og_settings_from_social_to_titles`. It got me totally confused as in: "wasn't the frontpage thing upgrade already done by Jimmy?" 🙂

Reply:
Will fix